crowbar_framework/vendor/assets/javascripts/jquery/ledUpdate.js

```diff
@@ -268,6 +268,12 @@
 
   LedUpdate.prototype.destroy = function() {
     $('.led.unready, .led.in_process, .led.spin').destroy();
+    // workaround https://bugzilla.suse.com/show_bug.cgi?id=949273
+    // to stop the green led from moving after node is in ready state
+    $('.led.ready').sprite({
```

Comment:
> not sure what that should bring

In case the same issue happens for other states than ready.



> but this happens already in L262 if it is what you mean.

Nope, that's not the same led states :-)